### ISSUE TITLE
Mumble disables my Bluetooth headset

### ISSUE THREAD

**Issue by Arthur-D** [mumble-voip/mumble#6280]
### Description

Whenever I start Mumble, my Bluetooth headset gets disabled system-wide.
It also applies whenever I have Mumble open first and then connect the headset, it connects but gets disabled as a playback device system-wide.
I can easily enable it again in my system sound settings, but it is rather annoying, and strange.

### Steps to reproduce

1. Connect Bluetooth headset.
2. Start Mumble.
3. See Bluetooth headset get disabled as playback device system-wide.
OR
1. Start Mumble.
2. Connect Bluetooth headset.
3. See Bluetooth headset is disabled as playback device system-wide.

### Mumble version

1.4.287

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

I am using OpenSUSE Tumbleweed, with PipeWire as sound server, under KDE Plasma. Mumble is installed as a Flatpak, though I don't know if that would really change anything in this case.

My Bluetooth headset is a Sony WH-XB910N, and if I don't have Mumble open, it gets set as default playback device as it should whenever I connect it. For the record, I have not found any other application messing with my system-wide playback like Mumble does, which is why I am filing this bug here.

### Screenshots

This is what happens whenever Mumble is open:
![image](https://github.com/mumble-voip/mumble/assets/5515261/25324b89-5d5e-4189-bb13-b038dcbc24f0)

Luckily, I can still enable it after the fact, Mumble doesn't hinder that, but the fact that it disables my Bluetooth headset has been going on for about a year now I think. This is how it should be:
![image](https://github.com/mumble-voip/mumble/assets/5515261/a6c21ea4-3551-4cec-b639-ab01d41c660d)

**Comment by Krzmbrzl** [comment]
Do you have the `Exclusive` Checkbox checked in the audio input and/or output settings?

**Comment by Arthur-D** [comment]
No, in fact that checkbox is greyed out for me in both Audio Input and Audio Output settings for Mumble.

(Also that is an amazingly quick response time, thanks for getting back to me.)

**Comment by davidebeatrici** [comment]
Does `journalctl` tell something related?

**Comment by Arthur-D** [comment]
I'm not sure, but maybe this is related? Doesn't really say anything about cause, but maybe it's something.
![image](https://github.com/mumble-voip/mumble/assets/5515261/aff8d22a-f026-40b7-affa-43e6a6b4528d)

**Comment by davidebeatrici** [comment]
Yep, looks like the PipeWire node goes into error...

**Comment by Arthur-D** [comment]
Is there something else I can do to help debug this? Or is this something that's not really a Mumble bug? Thank you very much for your time.

**Comment by Krzmbrzl** [comment]
It's likely a bug in our PipeWire implementation somehow. @davidebeatrici knows this best, so he might know what could be important here.
